source/features/release-download-count.css

```
@@ -4,6 +4,11 @@
 	min-width: 6em;
```

For the purpose of this bug fix, I think this is enough:

```suggestion
	min-width: 4em;
```

It can be improved further separately. Let's just test and merge